### DISCUSSION TITLE
How to access Action Logs? PostgreSQL in docker container?

### DISCUSSION THREAD

**Discussion by petrovivanvlad** [apache/superset#17413] [Q&A / Help]
Hello.

I need to make Action Logs pipeline into my data warehouse

Wanted to lurk inside of postgre, but when i type **"psql -U postgres"** inside of docker container, it says **"failed: FATAL:  role "root" does not exist"**

1. I wonder where can i get those logs, maybe it lies inside of this db?
2. If it exists in postgre, how can i connect to it?

Help

**Comment by kamalkeshavani-aiinside** [comment]
Check `SQLALCHEMY_DATABASE_URI` in your config file. This is where all the metadata of Superset is saved including action logs.
